platform/ethereum/blockbook/model.go

```diff
@@ -58,6 +58,7 @@ type TokenTransfer struct {
 
 // Token contains info about tokens held by an address
 type Token struct {
+	Balance  string               `json:"balance"`
```

Comment:
```suggestion
	Balance  string               `json:"balance,omitempty"`
```
this might be missing empty in some apis